### ISSUE TITLE
Merge supplementary alignment(s) with primary

### ISSUE THREAD

**Issue by niemasd** [niemasd/ViralMSA#14]
On some viral consensus sequences, e.g. ones with long spans of `N` in the middle, instead of outputting a single alignment, Minimap2 outputs a primary alignment (flag 0) and a supplementary alignment (flag 2048), e.g. the following:

```
@SQ	SN:MN908947.3	LN:29903
@PG	ID:minimap2	PN:minimap2	VN:2.17-r941	CL:minimap2 -t 8 --secondary=no --sam-hit-only -a -o OUTPUT.sam REFERENCE.fa.mmi INPUT.fas
original	0	MN908947.3	7326	60	7286S22371M	*	0	0	CTTTCGA...
original	2048	MN908947.3	40	60	6747M22910H	*	0	0	CTTTCGA...
```

I need to find a way to merge supplementary alignment(s), if any, with the primary alignment, e.g. as follows:

```
original	0	MN908947.3	40	60	29657M	*	0	0	CTTTCGA...
```

I created a GitHub Issue on the Minimap2 repo:

https://github.com/lh3/minimap2/issues/720

If there's no clean fix directly within Minimap2, before doing the conversion from SAM to FASTA, I'll need to parse the SAM to do this merge operation on all relevant sequences

**Comment by niemasd** [comment]
It seems like manually deleting `N`s from the sequence works as a temporary fix for now:

[viralmsa_out_delete_Ns.zip](https://github.com/niemasd/ViralMSA/files/6199210/viralmsa_out_delete_Ns.zip)

**Comment by niemasd** [comment]
Adding `--score-N=0` to the `minimap2` command fixed the issue (see https://github.com/niemasd/ViralMSA/commit/7e72dd7ad1a66646dd44e2551cbe862e572771f4). Fix incorporated into [ViralMSA 1.1.12](https://github.com/niemasd/ViralMSA/releases/tag/1.1.12)
